docs/howto/operate/new-hub/index.md

```diff
@@ -1,25 +1,75 @@
+  There are some cases where you must manually deploy or modify a hub.
+  See [](operate:manual-deploy) for more details.
 
-## Cloud provider-specific information
+## Overview of hub configuration
 
-The following sections contain information that is unique to each cloud provider.
-Follow these guides in addition to the more general instructions above.
+Many of our hubs are automatically deployed and updated using GitHub Workflows and configuration that is defined in [`pilot-hubs/config/hubs`](https://github.com/2i2c-org/pilot-hubs/tree/master/config/hubs).
+
+These are a collection of YAML files (one per cluster) that define the configuration for all of our hubs.
+To learn which subset of clusters are *automatically* deployed via GitHub workflows, inspect the `matrix:cluster_name:` list in [the `deploy-hubs.yaml` action](https://github.com/2i2c-org/pilot-hubs/blob/f2ffc8ef51427d5f824747917bfd51533daf3045/.github/workflows/deploy-hubs.yaml#L17-L31).
+
+The process of automatically updating and adding hubs is the same for all of the hubs deployed on these clusters.
+
+## To add a new hub
+
+To deploy a new hub, follow these steps:
+
+1. [Ask the Community Representative to fill in this issue template](https://github.com/2i2c-org/pilot-hubs/issues/new?assignees=&labels=type%3A+hub&template=2_new-hub.yml&title=New+Hub%3A+%3CHub+name%3E).
+   This will create a "New Hub" issue and ask the representative questions that are important for understanding how to set up their hub.
+   Once that issue is created, move to the next step.
+2. Decide whether you'll deploy on a pre-existing Kubernetes cluster, or if you'll need to create a new one.
+   See [](cluster:when-to-deploy) for information to help you decide.
+3. Determine the **hub template** that is needed.
+   Hub templates are pre-configured deployments for certain kinds of JupyterHubs.
+   There are a few base templates to choose from.
+   For more information about our templates and how to choose, see [](hub-templates).
+4. Add a configuration entry for your new hub.
+   Each entry is a Zero to JupyterHub configuration, and you can customize whatever you like.
+   The easiest way to add new configuration is to look at the entries for similar hubs in the same cluster YAML file, copy / paste one of them, and make modifications as needed for this specific hub.
+   For example, see the entries in [the 2i2c Google Cloud cluster configuration file](https://github.com/2i2c-org/pilot-hubs/blob/master/config/hubs/2i2c.cluster.yaml).
+   
+   :::{seealso}
+   See [](/topic/config.md) for more information about template configuration.
+   :::
+5. Create a Pull Request with the new hub entry, and get a team member to review it.
+6. Once you merge the pull request, the GitHub Workflow will detect that a new entry has been added to the configuration file.
+   It will then deploy a new JupyterHub with the configuration you've specified onto the corresponding cluster.
```

Comment:
I guess one additional step would be to "monitor" the deploy GH action to see if it run successfully and when you can start testing the new hub.

Reply:
good point - i added some extra steps to test the hub once it's up